### PR TITLE
Version 0.8.0

### PR DESCRIPTION
**Pull request by jbeezley**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.5.0'
+version = '0.8.0'
```
Comment (aashish24) [inline docs/conf.py]: Looks like we missed it for 0.7?

File: package.json [hunk 2]
```diff
@@ -49,5 +49,6 @@
     "gis",
     "webgl",
     "svg"
-  ]
+  ],
+  "main": "geo.js"
```
Comment (aashish24) [inline package.json]: curious why this is added

File: docs/conf.py (hunk 1 above)
Comment (jbeezley) [inline docs/conf.py]: Yes, our RTD site is badly neglected.

File: package.json (hunk 2 above)
Comment (jbeezley) [inline package.json]: It allows someone to use geojs with webpack or browserify more easily without having to provide an explicit [resolver](https://webpack.github.io/docs/configuration.html#resolve).

Comment (aashish24) [inline package.json]: ah, I see. thanks this helped.

**Comment by aashish24** [comment]
LGTM +2

**Pull request merged**